website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+
+When other steps must be taken to enable a feature or function, provide context in place of that feature so users can enable this feature.
+
+!!! Dont
+
+In this example, the button is disabled without context, leaving the user confused and unaware of the steps to enable the feature.
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing users from hovering, focusing, or clicking. For users with assistive technology, e.g., a screen reader, disabled elements (including their text) are almost entirely ignored.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+## Upgrading
+
+When upgradable features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity. Be mindful however to not overdue it and make the user feel overly advertised to.
+
+There are two ways to showcase an upgradable feature:
+
+- Space to explain
+- Interrupting an action
+
+### Space to explain
+
+In this example, "Terraform Health" is a an upgradable feature and is contextually relevant in the workspace. The UI has space to entice the user by explaining a bit of what this service can do for them.
+
+![Screen shot of Terraform workspace highlighting Terraform Health on the side bar below information.](/assets/patterns/disabled-patterns/upgrade-inline-example.png)
+
+### Interrupting an action
+
+In this example, a user is about to perform an action and notices something new within the list of options, one with a "Beta" badge. 
+
+![Showing a dropdown opened, with two list items. One for creating a static secret, while another using an auto-rotating secret with a beta badge next to it.](/assets/patterns/disabled-patterns/upgrade-modal-example-1.png)
+
+Enticed, they click it to use this feature, however, they are not enrolled in this tier and are interrupted by a modal explaining this feature a little more.
+
+![A modal explaining beta to the user, with a CTA to learn more, or close the modal.](/assets/patterns/disabled-patterns/upgrade-modal-example-2.png)
+
+Since the action is the only explanation a user has at the moment, providing a modal as an interruption to the action allows a user to understand more without fully committing. Only when a user wants to "learn more," they can then be lead to a page with more details.
```

Comment:
[suggestion] This might introduce a bit of brevity here

```suggestion
Interrupting the action with a modal helps a user understand more without fully committing, and if more details are necessary they can be directed to an informational page.
```
I'm not really sure what you're trying to say in the first part of this sentence ("Since the action is the only explanation..."), but this seems to read a bit more concise. WDYT?

Reply:
Your suggestion makes sense I'm adding it.